aea/cli/test.py

```diff
@@ -62,6 +62,8 @@
 COVERAGERC_CONFIG = """[run]
 omit =
     */tests/*
+    */*_pb2.py
+    */*_pb2_*.py
 
```

Comment:
example of effect on coverage measurement in open-autonomy abci connection
- before
    ```
    ---------- coverage: platform linux, python 3.10.4-final-0 -----------
    Name                                                                 Stmts   Miss  Cover
    ----------------------------------------------------------------------------------------
    packages/valory/connections/abci/__init__.py                             0      0   100%
    packages/valory/connections/abci/check_dependencies.py                  20      0   100%
    packages/valory/connections/abci/connection.py                         399      0   100%
    packages/valory/connections/abci/dialogues.py                           12      0   100%
    packages/valory/connections/abci/gogoproto/__init__.py                   0      0   100%
    packages/valory/connections/abci/gogoproto/gogo_pb2.py                 314      0   100%
    packages/valory/connections/abci/tendermint/__init__.py                  0      0   100%
    packages/valory/connections/abci/tendermint/abci/types_pb2.py          359      0   100%
    packages/valory/connections/abci/tendermint/abci/types_pb2_grpc.py     131     60    54%
    packages/valory/connections/abci/tendermint/crypto/keys_pb2.py          19      0   100%
    packages/valory/connections/abci/tendermint/crypto/proof_pb2.py         33      0   100%
    packages/valory/connections/abci/tendermint/types/params_pb2.py         47      0   100%
    packages/valory/connections/abci/tendermint/types/types_pb2.py         130      0   100%
    packages/valory/connections/abci/tendermint/types/validator_pb2.py      28      0   100%
    packages/valory/connections/abci/tendermint/version/types_pb2.py        19      0   100%
    packages/valory/connections/abci/tendermint_decoder.py                 126      0   100%
    packages/valory/connections/abci/tendermint_encoder.py                 191      0   100%
    ----------------------------------------------------------------------------------------
    TOTAL                                                                 1828     60    97%
    ```
- after
    ```
    ---------- coverage: platform linux, python 3.10.4-final-0 -----------
    Name                                                      Stmts   Miss  Cover
    -----------------------------------------------------------------------------
    packages/valory/connections/abci/__init__.py                  0      0   100%
    packages/valory/connections/abci/check_dependencies.py       20      0   100%
    packages/valory/connections/abci/connection.py              399      0   100%
    packages/valory/connections/abci/dialogues.py                12      0   100%
    packages/valory/connections/abci/gogoproto/__init__.py        0      0   100%
    packages/valory/connections/abci/tendermint/__init__.py       0      0   100%
    packages/valory/connections/abci/tendermint_decoder.py      126      0   100%
    packages/valory/connections/abci/tendermint_encoder.py      191      0   100%
    -----------------------------------------------------------------------------
    TOTAL                                                       748      0   100%
    ```

Reply:
suggest also adding https://github.com/valory-xyz/open-aea/blob/main/.coveragerc#L3 and